Bitbucket Server plugin: CHANGELOG & docs changes (#4862)
sourcegraph/sourcegraph: doc/admin/external_service/bitbucket_server.md

Bitbucket Server external service connections support the following configuration options, which are specified in the JSON editor in the site admin external services area.

<div markdown-func=jsonschemadoc jsonschemadoc:path="admin/external_service/bitbucket_server.schema.json">[View page on docs.sourcegraph.com](https://docs.sourcegraph.com/admin/external_service/bitbucket_server) to see rendered content.</div>

## Native extension

> For production usage, we recommend installing the Sourcegraph Bitbucket Server plugin for all users (so that each user doesn't need to install the browser extension individually). This involves adding a new extension to the extension directory of your Phabricator instance.

Reviewer: ```suggestion
For production usage, we recommend installing the Sourcegraph Bitbucket Server plugin for all users (so that each user doesn't need to install and configure the browser extension individually). This involves adding a new add-on to  your Bitbucket Server instance.
```

Author: Oops. Thanks for catching! I obviously didn't do a thorough enough jobs of adapting those Phabricator docs 🙂
